.gitignore

```diff
@@ -5,3 +5,5 @@ nimcache
 !tests/image.jpg
 tests/t*
 !tests/t*.nim
+examples/*
+!examples/*.*
```

Comment:
this gonna unignore `exe`'s on windows. better match `.nim` and sample file explicitly

Reply:
thankx, I fixed

Reply:
Ah, sorry, i fix more

Reply:
Change to

```
examples/*
!examples/*.nim
!examples/*.nims
!examples/sample.png
```